src/main/cms/cms_menu_blackbox.c

```diff
@@ -243,6 +261,25 @@ static CMS_Menu cmsx_menuEraseFlashCheck = {
     .onDisplayUpdate = NULL,
     .entries = menuEraseFlashCheckEntries
 };
+
+static const OSD_Entry menuStorageDeviceCheckEntries[] = {
+    { "CONFIRM STORAGE", OME_Label, NULL, NULL},
+    { "YES",            OME_Funcall, cmsx_StorageDevice, NULL },
```

Comment:
Maybe `yes` should be second entry (so that it is not selected when entering menu?)

Reply:
Agree, this will more safety. And Erase action need second entry too.